sormas-api/src/test/java/de/symeda/sormas/api/utils/DataHelperTest.java

```diff
@@ -133,6 +134,7 @@ public void testParseDateWithExceptionForDeFormat() throws ParseException {
 	}
 
 	@Test
+	@Ignore
```

Comment:
We should remove/resolve the `@Ignore` before merging

Reply:
true

Reply:
removed